scripts/ci/sync.sh

```diff
@@ -40,6 +40,10 @@ destination_bucket_uri="s3://${destination_bucket}"
 # created in another account, in which case subsequent operations on the bucket will also
 # fail, causing this script to exit nonzero. In either case, it's okay to continue.
 aws s3 mb $destination_bucket_uri --region "$(aws_region)" || true
+aws s3api put-public-access-block --bucket $destination_bucket --public-access-block-configuration BlockPublicAcls=false
+aws s3api put-bucket-ownership-controls --bucket $destination_bucket --ownership-controls="Rules=[{ObjectOwnership=ObjectWriter}]"
```

Comment:
this had to be added because if not the line below will fail with
```Bucket cannot have ACLs set with ObjectOwnership's BucketOwnerEnforced setting```